### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by CodeMaster7000**

### CODE REVIEW

**Comment by SmashedFrenzy16** [comment]
This is copying some of the changes that I made in PR #3639

**Comment by SmashedFrenzy16** [comment]
So this PR is not necessary

**Comment by SmashedFrenzy16** [comment]
At least, for the `__init__.py`, as that is not needed.

**Comment by CodeMaster7000** [comment]
@SmashedFrenzy16 The phrasing is different from that in your PR. An additional message is also included.

**Comment by SmashedFrenzy16** [comment]
@CodeMaster7000 you could have told me to update my PR instead. Anyways, you could always delete that file.

**Comment by CodeMaster7000** [comment]
@SmashedFrenzy16 I honestly think it is fine for now. Once the change has been made, Pygame will most likely close or merge the PRs anyways.

**Comment by SmashedFrenzy16** [comment]
@CodeMaster7000 is cannot be fine that you copy other people's pull requests to only get your merged. The best solution would be to delete the `__init__.py` file because you made unecessary extra commits in that change.

**Review by SmashedFrenzy16 — CHANGES_REQUESTED** [review]
Delete `__init__.py` from this PR as it is not necessary.

**Comment by novialriptide** [comment]
> Delete `__init__.py` as it is not necessary.

Deleting the file would break pygame

**Comment by SmashedFrenzy16** [comment]
No I mean delete it from this PR

File: src_py/__init__.py [hunk 1]
```diff
@@ -339,7 +339,7 @@ def __color_reduce(c):
             ver, *get_sdl_version() + sys.version_info[0:3]
         )
     )
-    print("Hello from the pygame community. https://www.pygame.org/contribute.html")
+    print("Hello from the Pygame community! To contribute, visit https://www.pygame.org/contribute.html. We appreciate you taking the time to do so.")
```
Comment (novialriptide) [inline src_py/__init__.py:342]: Lowercase p is usually preferred when referring to "pygame" as per PEP8
https://peps.python.org/pep-0008/#package-and-module-names
image

**Comment by oddbookworm** [comment]
@SmashedFrenzy16 @CodeMaster7000 the comments of a pull request is not an appropriate place for you two to argue about whose pull request should contain which changes. If you have a disagreement about something, please sort that out in private. Nobody reviewing your PR's wants to sort through your argument to see if there's any important information there. Also, if someone reviews your PR and it's clear changes to the PR are needed, you won't get a merge unless you fix them. Same goes for failing actions (like the lint check, which means you need to run `python setup.py format`)

**Comment by CodeMaster7000** [comment]
I understand and I apologise. It might have got a bit out of hand with 2 PRs having overlapping changes.

**Comment by SmashedFrenzy16** [comment]
I also understand and apologise.

**Comment by Starbuck5** [comment]
Hey @SmashedFrenzy16 and @CodeMaster7000, I believe these changes make the content slightly worse. I talked to some other contributors and I believe we're in agreement about this, so I'm going to close this and #3639 (similar content)

You seem like you want to help out here, which is admirable. If you'd like to tackle an issue (perhaps you can work together on this, you seem to know each other), #3357 could be good. Alternatively, you could try to enhance the docs. I think one promising area would be function names that are written out instead of linked. If some function description references another function it should be possible to click on that and go to the other function, but I don't believe this is the case in all cases.

**Pull request closed without merging**